File: Programming-Basics-With-JavaScript/Exams/01-Old-Exams/01-PB-Exam-June-2019/01.seriesCalculator.js
function seriesCalculator(input) {

    const seriesName = input[0];
    const seasonsCount = Number(input[1]);
    const episodesCount = Number(input[2]);
    const episodeDuration = Number(input[3]);

    // Във всеки епизод има реклами, които са с продължителност 20% от времето на един епизод.
    const advertising = 0.2 * episodeDuration;
    const episodeDurationPlusAdvertising = episodeDuration + advertising;
    // Всеки сезон завършва със специален епизод, който е с 10м по-дълъг от обичайното.
    const extraTime = seasonsCount * 10;
    // Времето, необходимо за изглеждане на всички епизоди, закръглено до най-близкото цяло число надолу.
    const totalTime = Math.floor(episodeDurationPlusAdvertising * episodesCount * seasonsCount + extraTime);

    console.log(`Total time needed to watch the ${seriesName} series is ${totalTime} minutes.`);
}
seriesCalculator(["Lucifer",
    "3",
    "18",
    "55"]);

File: Python_Basics/Exams/07.exam_preparation_jun_2023/01.series_calculator.py
series_name = input()
seasons_count = int(input())
episodes_count = int(input())
ad_free_duration_per_episode = float(input())

#  Във всеки епизод има реклами, които са с продължителност 20% от времето на един епизод.
ads_duration = ad_free_duration_per_episode * 0.2

#  Също така знаете, че всеки сезон завършва със специален епизод, който е с 10м по-дълъг от обичайното.
last_episode_bonus_duration = 10

total_time = seasons_count * (episodes_count * (ad_free_duration_per_episode + ads_duration) + last_episode_bonus_duration)

print(f'Total time needed to watch the {series_name} series is {int(total_time)} minutes.')
